modules/fastp/meta.yml

```diff
@@ -21,7 +21,8 @@ input:
       type: file
       description: |
         List of input FastQ files of size 1 and 2 for single-end and paired-end data,
-        respectively.
+        respectively. If you wish to run interleaved paired-end, single-end data
+        but with `--interleaved_in` in your `modules.conf`'s `ext.args` for the module.
```

Comment:
I don't understand this sentence

Reply:
missing words, my bad...

Reply:
@FriederikeHanssen please check hte updated version :)